types/wizard.d.ts

```diff
@@ -8,10 +8,13 @@ declare global {
   namespace LHCI {
     namespace WizardCommand {
       export interface Options {
-        wizard?: 'new-project' | 'reset-admin-token';
+        wizard?: 'new-project' | 'reset-admin-token' | 'reset-build-token';
```

Comment:
I think this was missing in the type definition

Reply:
thanks 👍